### DISCUSSION TITLE
Unable to Add Page Links

### DISCUSSION THREAD

**Discussion by ZachHandley** [JulianPrieber/littlelink-custom#355] [Issues]
https://i.imgur.com/swg2B7l.png as title says I can't add page links, just wants me to login. It also says that my database is accessible from the web (which it is) but I do not know how to fix it, given that I already set it to not be read/writeable from the public but it still says that and trying to google for it is yielding no results

Platform: `Ubuntu 22.04 LTS`
Server type: `NGINX`
PHP version: `8.2`

[//]: # (Please make sure you have read the system requirements at https://llc.ovh/req before submitting your issue.)

**Comment by JulianPrieber** [comment]
I can't really look into this right now, I'm currently not home.

We have documentation for NGINX:
https://docs.littlelink-custom.com/getting-started/other-webservers/

**Please** add this to your NGINX block to protect your files:

```
location ~ ^\. {
deny all;
}

location ~ \.sqlite$ {
deny all;
}

location ~ \.env$ {
deny all;
}

location ~ /\.htaccess {
allow all;
}
```

**Comment by ZachHandley** [comment]
That fixed the database error thank you, I'll check on the block, maybe it's Laravel thinking I'm not logged in for some reason

**Comment by JulianPrieber** [comment]
@ZachHandley Have you been able to resolve your issue, or have any news on this?

**Comment by JulianPrieber** [comment]
![swg2B7l](https://user-images.githubusercontent.com/60265788/220618771-e35a3013-35a1-4c7e-9e61-7f3af1472b7b.png)
The image here confuses me a bit, some elements are not rendering as they should. Do you have any console errors in your browser?

**Comment by ZachHandley** [comment]
@JulianPrieber Currently I can't login because I disabled registration, so I'll get back to you when I remember how to fix that or reinstall it tonight

**Comment by ZachHandley** [comment]
Okay so logged in, nothing is in the console

**Comment by JulianPrieber** [comment]
Do these elements still look like this?
![220618771-e35a3013-35a1-4c7e-9e61-7f3af1472b7b](https://user-images.githubusercontent.com/60265788/220778665-eb034717-08f2-4cd1-841f-ac5af78cf066.png)


For reference, this is what It's supposed to look like:
![chrome_cMIqcw2wqK](https://user-images.githubusercontent.com/60265788/220778823-14111c84-44a4-4e27-a23f-b7a02dd43c94.png)

**Comment by ZachHandley** [comment]
https://i.imgur.com/2g5TNES.png this is what it looks like

**Comment by JulianPrieber** [comment]
Hey @ZachHandley were you able to fix your issue? Any updates?

**Comment by 0x4E4448** [comment]
I am experiencing this same issue. I'm running on Kubernetes using nginx-ingress-controller.

AH01797: client denied by server configuration: /htdocs/.env
AH01797: client denied by server configuration: /htdocs/database/database.sqlite

My understanding is that nginx-ingress-controller already sets the X-Forwarded-For and X-Forwarded-Proto headers. I am seeing in /panel/phpinfo that HTTP_X_FORWARDED_FOR is set to the IP of my load balancer and HTTP_X_FORWARED_PROTO is set to https.

**Comment by ZachHandley** [comment]
@JulianPrieber After updating my Nginx Configuration still getting the same issue

Also getting a 
```
Uncaught ReferenceError: $ is not defined
    <anonymous> https://zachhandley.com/:14864
[zachhandley.com:14864:1](https://zachhandley.com/)
``` 

on the home screen at line 

```
$('#share-share-button').on('click', () => {
  if (navigator.share) {
    navigator.share({
        title: '',
        text: '',
        url: '',
      })
      .then(() => console.log('Successful share'))
      .catch((error) => console.log('Error sharing', error));
  } else {
    console.log('Share not supported on this browser, do it the old way.');
  }
});
```
